native/proto/src/proto/operator.proto

```diff
@@ -52,6 +52,7 @@ message SparkPartitionedFile {
   int64 start = 2;
   int64 length = 3;
   int64 file_size = 4;
+  repeated spark.spark_expression.Expr partition_values = 5;
```

Comment:
aren't the partition values just strings?

Reply:
No. Although for Hive partitioned table, partition values are dictionary names which are strings, but once Spark reads these strings back, they are casted to corresponding data types of partition columns.

Reply:
Ah makes sense.